website/src/content/deployment/compile-custom-application.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Compiling Custom Application
```

Comment:
PS: you also need to update the navigation data yaml file

Reply:
Filename and header are still different though. Is that intended?